service/notifier.go

```diff
@@ -82,12 +82,17 @@ func (n *Notifier) ProcessWebhook(b *models.WebhookBody) error {
 		}
 		return err
 	case "pr:comment:added":
-		return n.prCommentAdded(b)
+		return prCommentAdded(n.Slack.GetUserByEmail, n.Slack.PostMessage, n.Bitbucket.GetPrActivity, b)
 	}
 	return nil
 }
 
-func (n *Notifier) prCommentAdded(b *models.WebhookBody) error {
+func prCommentAdded(
```

Comment:
I refactored this in order to test it easier (I didnt manage to properly mock all the Notifer model).

It uses simple dependency injection by passing functions as arguments. Its valid in typescript world, but I dont know if there is a better way of doing this in GO.